README.md

```diff
@@ -6,3 +6,19 @@ This paper investigates how we can model some of the behavioural gaming that
 takes place between the Emergency Department (ED) and the Emergency Medical
 Services (EMS). A 3-player game is investigated that is informed by a queueing
 theoretic model.
+
+Contents
+-----------------------------------------------------------------------
+1. Introduction - Motivation
+2. Overview of game theoretic model
+3. A queueing models with 2 consecutive buffer centres
+   1. Performance Measures
+   2. Example
+4. Behavioural Methodology
+   1. Backwards Induction
+   2. Nash Equilibrium
+   3. Learning Algorithms
+5. EMS - ED application
+   1. Application
+   2. Data Analysis
+6. Conclusion
```

Comment:
I'd suggest you remove this, pop it in an issue.

Only because we risk changing the structure in the paper at some point but not changing it in the README.